.github/workflows/test-phpunit.yml

```diff
@@ -0,0 +1,108 @@
+name: PHPUnit
+
+on:
+  push:
```

Comment:
I'm not a fan of the `push` triggers, since anything pushed will already have been tested during PRs. I'm fine if we want to leave them as a "just in case" but it will likely just end up being debris in the Actions tab.